doc/api/n-api.md

```diff
@@ -697,6 +697,58 @@ functions) that can be passed as parameters when `js_callback` is invoked. This
 pointer is managed entirely by the threads and this callback. Thus this callback
 should free the data.
 
+#### napi_key_collection_mode
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+todo
+
+```C
+typedef enum {
+  napi_key_include_all_properties,
+  napi_key_include_own_properties
```

Comment:
Would there be any chance to not include own properties? If not, would `napi_key_own_only` be sufficient to indicate own properties only collection mode?

Apologies if my previous review makes any wrong impression.